### ISSUE TITLE
Suggest a new design

### ISSUE THREAD

**Issue by DavidEBest** [codefordayton/npverify#3]
There is only minimal styling on the application. We welcome recommendations on ways to improve it.

The live site is at [NPVerify.codefordayton.org](https://npverify.codefordayton.org/). We will happily accept designs that flesh out the look and feel of the site.

To submit a design:

For non-code solutions:

- [ ] Create a `design` folder in your pull request that contains design artifacts. These design artifacts could include screenshots, Adobe XD files, or files from whatever design tool you prefer.

For code solutions:

- [ ] Update the templates to use appropriate HTML and CSS. Be aware that there is an additional issue (#2) to switch from Bootstrap to Tailwind.

We do not have a solid vision for what the site should look like, but we will be accepting of reasonable suggestions.

Here are some sites in similar niches or areas:

- [Code For Dayton](https://www.codefordayton.org/)
- [Dayton Serves](https://daytonserves.org/)
- [Volunteer Match](https://www.volunteermatch.org/)

Feel free to ask questions on the issue!

**Comment by maebasmrt** [comment]
Hi, @DavidEBest I would love to work on this, please assign it to me.

**Comment by DavidEBest** [comment]
@maebasmrt Absolutely! Please let me know if you you need additional information about this.

**Comment by maebasmrt** [comment]
Ok David, I will

**Comment by maebasmrt** [comment]
Howdy, David

Just to update you, I am currently working on the styleguide for the project

**Comment by DavidEBest** [comment]
Sounds great! Thanks!
